app/settings.py

```diff
@@ -7,6 +7,9 @@
 MAX_UPLOADS_PER_HOUR = 100
 MAX_UPLOADS_PER_MINUTE = 20
 ALLOWED_ORIGINS = ["*"]
+NAME_STRATEGY_RANDOMSTR = 'randomstr'
+NAME_STRATEGY_UUIDV4 = 'uuidv4'
```

Comment:
I see why you do this but for consistency's sake I would prefer that you just compare this against the strings in main instead of declaring them as constants here.
The settings.py should act as an overview of which params can be set and these constants don't belong in it.

Reply:
Done :-)

Reply:
Thank you!